### ISSUE TITLE
Redux vs Context API

### ISSUE THREAD

**Issue by berniedurfee** [Lambda-School-Labs/labs-engineering-standards#12]
Please discuss...

**Comment by berniedurfee** [comment]
I think this is a toss-up, I don't see a reason to restrict the use of Redux

**Comment by ryan-hamblin** [comment]
BOTH!. I don't see a reason to restrict Redux, it's still extremely popular, but yes, Context and Hooks + useReducer are the new hottness

**Comment by tinomen** [comment]
+1 encourage context/hooks

**Comment by berniedurfee** [comment]
> BOTH!. I don't see a reason to restrict Redux, it's still extremely popular, but yes, Context and Hooks + useReducer are the new hottness

@ryan-hamblin I think we need a React topic, as there will be a bunch of things related to React that we'll want to set standards for. Can you create a new topic and add in policies for state management? Many 'Approved State Management' frameworks or something.

**Comment by ericlugo** [comment]
Just saw this while in our Labs lecture courtesy of Jake. Figured I'd post a quick comment since it looks like students can join the discussion?

From an old comment by [Sebastian Markbage](https://github.com/facebook/react/issues/14110):
> My personal summary is that new context is ready to be used for low frequency unlikely updates (like locale/theme). It's also good to use it in the same way as old context was used. I.e. for static values and then propagate updates through subscriptions. It's not ready to be used as a replacement for all Flux-like state propagation.

As far as I've been able to dig, it still seems like Context is not all-encompassing. To be fair, what I'm quoting is over a year old (so an eternity-and-a-half in trendy tool minutes) and it seems like they're making strides in how much it can be used for, but the focus of the project doesn't seem to be to compete with state management frameworks like Redux.
